docs/users.rst

```diff
@@ -224,3 +224,16 @@ as plugins:
 
   * If Pulp integration is enabled, delete the image from the
     container registry
+
+Arrangement version 2 (orchestrator creates filesystem)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This arrangement is identical to version 1 except that the
+add_filesystem plugin runs in the orchestrator build as well. This is
+to create a single Koji "image-build" task to create filesystems for
```

Comment:
Might be nice to make `"image-build"` a link to https://docs.pagure.org/koji/image_build/